src/parse/asp/config.go

```diff
@@ -136,6 +136,9 @@ func getConfigKey(aspKey, configKey string) string {
 
 // pluginConfig loads the plugin's config into a pyDict. It will load con
 func pluginConfig(pluginState *core.BuildState, pkgState *core.BuildState) pyDict {
+	log.Debug("pluginState %s", pluginState.CurrentSubrepo)
```

Comment:
Just to check, did you intend to leave these debug logs here?

Reply:
good spot thanks, no I definitely did not! forgot I had some local changes floating around before :)